src/Styling/Home.css

```diff
@@ -1,7 +1,75 @@
+/* font styles */
+.style_1 {
+  font-family: 'Poppins';
+  font-weight: 600;
+  font-size: 6.25rem;
+  color: #DE5B31;
+}
+#style_1 {
```

Comment:
I would refrain from using the same style name for two styles. Even if one is a className and the other is an ID. It could cause confusion down the road. You could try something like this for the ID instead:
"#style_1_transparent"